circuit/src/poseidon/params/mod.rs

```diff
@@ -4,22 +4,27 @@ pub mod bn254_5x5;
 use halo2wrong::halo2::{arithmetic::FieldExt, plonk::Expression};
 
 pub trait RoundParams<F: FieldExt, const WIDTH: usize>: Sbox {
+	/// Returns integer 8 as usize.
```

Comment:
This is just a trait definitions.
For every parameter set there will be a different implementation.
So, let's just say that this function should return a number of full rounds.